protos/camera_server/camera_server.proto

```diff
@@ -58,6 +85,55 @@ message StopVideoResponse {
     int32 stream_id = 1;    // video stream id
 }
 
+message SubscribeStartVideoStreamingRequest {}
+message StartVideoStreamingResponse {
+    int32 stream_id = 1;    // video stream id
+}
```

Comment:
Shouldn't this also include a result? Otherwise, what if the video stream starting failed?